java/libphonenumber/pom.xml

```diff
@@ -30,6 +30,24 @@
     </testResources>
     <plugins>
       <plugin>
+        <groupId>org.apache.felix</groupId>
+        <artifactId>maven-bundle-plugin</artifactId>
```

Comment:
Please add version number here and other plugins as well to keep the Maven build hermetic.